lib/pure/times.nim

```diff
@@ -590,10 +590,28 @@ proc fromUnix*(unix: int64): Time
 
 proc toUnix*(t: Time): int64 {.benign, tags: [], raises: [], noSideEffect.} =
   ## Convert ``t`` to a unix timestamp (seconds since ``1970-01-01T00:00:00Z``).
+  ## See also `toUnix`.
   runnableExamples:
     doAssert fromUnix(0).toUnix() == 0
   t.seconds
 
+proc fromUnix*(unix: float): Time
```

Comment:
Name it ``fromUnixFloat`` for consistency. Also: Time in Unix is not a float so this is weird.

Reply:
fixed